.github/workflows/preview.yml

```diff
@@ -35,3 +35,13 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           AmplifyAppId: ${{ secrets.AMPLIFY_APP_ID }}
           AWS_REGION: 'us-east-1'
+      - name: comment with preview url
```

Comment:
Is there a way to make this run only once instead of on every push? Maybe we could check whether the trigger type is "labeled"?